common/thorhelper/thorcommon.cpp

```diff
@@ -2043,7 +2043,8 @@ static bool getTranslators(Owned<const IDynamicTransform> &translator, Owned<con
             }
         }
 
-        //MORE: Could avoid creating translator if (projectedCrc != sourceCrc)
+        //MORE: Could avoid creating translator if (projectedCrc != sourceCrc).
+        //However that would allow false negatives if the format crcs happened to match, leading to crashes on untranslated files.
```

Comment:
The same crashes as people would get today...

I think it's sufficiently unlikely that we should ignore the possibility.